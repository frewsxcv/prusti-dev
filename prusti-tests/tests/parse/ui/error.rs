use prusti_contracts::*;

#[requires(true)]
#[ensures(true)]
fn has_an_error() {
    incorrect
}

fn main() {}
