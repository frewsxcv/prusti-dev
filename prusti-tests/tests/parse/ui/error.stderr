error[E0425]: cannot find value `incorrect` in this scope
 --> $DIR/error.rs:6:5
  |
6 |     incorrect
  |     ^^^^^^^^^ not found in this scope

error: aborting due to previous error

For more information about this error, try `rustc --explain E0425`.
